pkg/build/build_test.go

```diff
@@ -0,0 +1,92 @@
+package build
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"github.com/suse-edge/edge-image-builder/pkg/config"
+)
+
+func TestPrepareBuildDir(t *testing.T) {
+	// Setup
+	bc := config.BuildConfig{}
+	builder := New(nil, &bc)
```

Comment:
Nit: This is an example where instantiating looks much cleaner. Doesn't need to happen in this PR but it can look like:
```
	builder := Builder{
		buildConfig: &config.BuildConfig{},
	}
```